proposal.rst

```diff
@@ -67,19 +67,22 @@ FILLME WITH THE TITLE OF YOUR PROPOSAL
 About You
 =========
 
- * Name: FILLME
+ * Name: Prithi Pal Singh
 
- * E-mail address: FILLME
+ * E-mail address: prithipal.singh285@gmail.com
 
- * IRC nickname(s): FILLME
+ * IRC nickname(s): jugnu
 
- * Any personal websites, blogs, social media, etc: FILLME
-
- * github URL: FILLME
+ * Any personal websites, blogs, social media, etc: LinkedIn : www.linkedin.com/in/prithipal-285
+ * github URL: github.com/PrithiPal
 
  * Biography:
 
-.. Tell us a bit about yourself.
+Hello to you people reading this. I am Prithi, 5th BSc. Computer science undergrad @ Simon Fraser University, Burnaby, BC Canada. Talking about my academic experience, there are couple of courses that I enjoyed includes ones on Comparative Programming Languages (Go,Haskell,Clojure), NLP, Computer Vision, Compiler Design and why not Machine learning. On top of the school, I participated in two Co-ops with data-centric roles encompassing analytics and back-end building. In school, I also hold onto two different research assistant positions in Bioinformatics and Computational Linguistics for Reddit Sub-thread where I proposed and implemented solutions as per the requirements. I guess My linked will summarize stuff better. 
```

Comment:
I'm really struggling to work out quite what this might mean:

> I also hold onto two different research assistant positions in Bioinformatics and Computational Linguistics for Reddit Sub-thread where I proposed and implemented solutions as per the requirements

Did two different sentences accidentally get spliced together?

Sadly linkedin seem to insist you create an account before you can view anything, so even if it does summarise better that doesn't really help me.